File: framework/languages/java/e2e/SpotService/Client/src/main/java/systems/zlink/e2e/spotservice/client/Scenarios/SmF1Scenario.java
package systems.zlink.e2e.spotservice.client.Scenarios;

import systems.zlink.e2e.spotservice.client.Support.GatewayScenarioClient;

public final class SmF1Scenario {
    private SmF1Scenario() {
    }

    public static void run(GatewayScenarioClient gateway) {
        gateway.runMode("route-mesh");
    }
}

File: framework/languages/java/e2e-kotlin/SpotService/Client/src/main/kotlin/systems/zlink/e2e/kotlin/spotservice/client/scenarios/SmF1Scenario.kt
package systems.zlink.e2e.kotlin.spotservice.client.scenarios

import systems.zlink.contracts.core.RoutingId
import systems.zlink.e2e.kotlin.spotservice.Contracts
import systems.zlink.framework.spots.ZLinkSpotOutbound

internal object SmF1Scenario {
    fun run(outbound: ZLinkSpotOutbound) {
        outbound.sendToSpot(RoutingId.from("room-a"), Contracts.StateCommand("mixed-route-send"))
            .await()
        println("scenario SM-F1 passed")
    }
}
